Route/PathInfoBuilderSlashes.php

```diff
@@ -26,11 +26,11 @@ public function build(AdminInterface $admin, RouteCollection $collection)
     {
         $collection->add('list');
         $collection->add('create');
-        $collection->add('batch', null, array(), array('_method' => 'POST'));
+        $collection->add('batch', null, array());
```

Comment:
There is [a dedicated argument for methods](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Route/RouteCollection.php#L70), please use it.